### DISCUSSION TITLE
"No module named 'packaging.metadata'" after installing 1.8.2 with Conda

### DISCUSSION THREAD

**Discussion by l0b0** [python-poetry/poetry#9400] [General] [answered]
Issue #9034 seems to be still happening in 1.8.2 when installed with Conda. [Relevant PR](https://github.com/linz/emergency-management-tools/pull/207/files).

Commands:

```bash
conda install --channel=conda-forge --quiet --yes gdal=3.6.4 poetry=1.8.2
poetry install --only=main --no-root
poetry run jupyter nbconvert --debug --execute --inplace --to=notebook sentinel1_water_extraction.ipynb
```

Log of the last command:

```console
No module named 'packaging.metadata'
```

Update: Tried to fix by [installing the very latest (1.8.3, not yet available from Conda) with pip instead of Conda](https://github.com/linz/emergency-management-tools/pull/341/files), but I'm getting the same error. The `pip` log even says it's installing packaging 24.0:

```text
Collecting packaging>=23.1 (from poetry==1.8.3)
  Downloading packaging-24.0-py3-none-any.whl.metadata (3.2 kB)
```

**Comment by Secrus** [comment]
Sounds like a Conda (which we don't maintain) quirk. We properly specify the required packaging version in our metadata.

**Comment by l0b0** [comment]
FYI, [conda-forge has labelled an older build of Poetry 1.8.3 as "broken" on OSX and Linux](https://anaconda.org/conda-forge/poetry/files). There doesn't seem to be any details beyond that, unfortunately.

**Reply by dimbleby** [reply]
I think that's basically an acknowledgement that they got something wrong, there are now newer 1.8.3 conda poetry things.  So I guess they fixed something at their end.

**Reply by l0b0** [reply, marked as answer]
The newer ones don't work either, AFAICT. [Reported upstream](https://github.com/conda-forge/poetry-feedstock/issues/101).
